src/test/kotlin/com/example/delivery/service/ProductServiceTest.kt

```diff
@@ -0,0 +1,81 @@
+package com.example.delivery.service
+
+import com.example.delivery.ProductFixture.createProductDTO
+import com.example.delivery.ProductFixture.domainProduct
+import com.example.delivery.ProductFixture.product
+import com.example.delivery.exception.NotFoundException
+import com.example.delivery.repository.ProductRepository
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.api.extension.ExtendWith
+import org.mockito.InjectMocks
+import org.mockito.Mock
+import org.mockito.Mockito
+import org.mockito.junit.jupiter.MockitoExtension
+import org.mockito.kotlin.times
+import org.mockito.kotlin.verify
+import kotlin.test.assertEquals
+
+@ExtendWith(MockitoExtension::class)
+internal class ProductServiceTest {
+    @Mock
+    lateinit var productRepository: ProductRepository
+
+    @InjectMocks
+    lateinit var productService: ProductService
+
+
+    @Test
+    @DisplayName("Add valid product")
```

Comment:
```suggestion
```

why do we need `@DisplayName`? The name of the test function is pretty clear.

In most cases `@DisplayName` is used in Java, because in java we don't have function names like
```kotlin
fun `some function name`()
```